README.md

```diff
@@ -1,63 +1,28 @@
 # Swiss Post Design System
 
-![Swiss Post Design System splash screen](https://github.com/swisspost/design-system/assets/1659006/c11afbac-8a71-4416-ae3c-ec87f4e10412)
+![Swiss Post Design System splash screen](https://github.com/swisspost/design-system/assets/1659006/e84f1fea-e666-4853-8c85-726a6bf22e6c)
 
 The Swiss Post Design System pattern library for a unified and accessible user experience across the web platform.
 
+## Documentation
 
-## Packages
-
-### Styles
-
-[![npm](https://img.shields.io/npm/v/@swisspost/design-system-styles)](https://www.npmjs.com/package/@swisspost/design-system-styles)
-
-[Documentation](https://design-system.post.ch) · [Changelog](/packages/styles/CHANGELOG.md)
-
-The styling package, including theming for [Bootstrap](https://getbootstrap.com/) and [ng-bootstrap](https://ng-bootstrap.github.io/#/home) components.
-
-```bash
-npm install @swisspost/design-system-styles
-```
-
-### Internet Header
-
-[![npm](https://img.shields.io/npm/v/@swisspost/internet-header)](https://www.npmjs.com/package/@swisspost/internet-header)
-
-[Documentation](https://next.design-system.post.ch/?path=/docs/internet-header-getting-started--docs) · [Changelog](/packages/internet-header/CHANGELOG.md)
-
-The header for client facing applications.
+[Current documentation](https://design-system.post.ch/#/home) - Contains `ng-bootstrap` components
```

Comment:
I don't have anything better as a proposition, but currently it could be read as "Contains **Only** `ng-bootstrap` components", so it might be confusing.

Reply:
Updated the description